src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -482,13 +484,21 @@ export default class ResultsViewPage extends React.Component<IResultsViewPagePro
     }
 
     public render() {
-        return (
-            <PageLayout noMargin={true}>
-                {
-                    this.pageContent
-                }
-            </PageLayout>
-        )
+
```

Comment:
we should probably handle studies loading here, but we don't.  don't really want to worry about it now